bitbucket-pipelines.yml

```diff
@@ -14,7 +14,8 @@ definitions:
         script:
           - autoreconf --install
           - ./configure
-          - export SONAR_SCANNER_VERSION=4.4.0.2170
+          - export SONAR_SCANNER_VERSION=4.4.0.2170 # Find the latest version in the "Linux" link on this page:
+                                                    # https://sonarcloud.io/documentation/analysis/scan/sonarscanner/
           - export SONAR_SCANNER_HOME=$HOME/.sonar/sonar-scanner-$SONAR_SCANNER_VERSION-linux
           - export BW_OUTPUT=$HOME/.sonar/bw-output
```

Comment:
Is there a way to extract the environment variable definitions outside of the script into a separate yml block, like [in gh-actions](https://github.com/sc-sq-c-family-examples/macos-xcode-gh-actions-sc/blob/main/.github/workflows/build.yml#L12)?